src/Player/Player.tsx

```diff
@@ -33,31 +35,30 @@ const Player: React.FC<Props> = ({ currentRecord }) => {
     })
   }, [websocket])
 
-  useEffect(() => {
-    if (!record || !player) {
-      return
-    }
-    const now = moment()
-    const offset = moment.duration(now.diff(record.playedAt))
-
-    player.seekTo(offset.as('seconds'))
-  }, [player, record])
-
   if (!record) {
     return <p>Nothing Playing!</p>
   }
 
-  const setRefFromPlayer = (player: ReactPlayer): void => setPlayer(player)
+  const changeProgress = (opts: { played: number }): void => {
+    setProgress(opts.played * 100)
+  }
+  const changeVolume = (ev: React.ChangeEvent<HTMLInputElement>): void => {
+    setVolume(parseFloat(ev.currentTarget.value))
+  }
+
   return (
-    <ReactPlayer
-      ref={setRefFromPlayer}
-      url={`https://www.youtube.com/watch?v=${record.song.youtubeId}`}
-      controls={true}
-      playing={true}
-      volume={0}
-      height="100%"
-      width="100%"
-    />
+    <>
+      <PlayerPrimitive
+        changeProgress={changeProgress}
+        playedAt={record.playedAt}
+        youtubeId={record.song.youtubeId}
+        volume={volume}
+      />
+      <Box width="100%" height="6px">
+        <Box width={`${progress}%`} height="100%" bg="text" />
+      </Box>
+      <input onChange={changeVolume} type="range" min="0" max="1" step=".01" />
```

Comment:
Maybe use this [Slider](https://rebassjs.org/forms/slider) component here?

Reply:
Good call.  This sent me down a slight rabbit hole because apparently `Slider` is not yet included in the type definitions for react forms.  See https://github.com/go-between/musicbox/pull/28/commits/9d61e5ccadddedc742ad313fc7324b0b7b5c80ca for our workaround for now.

<img width="876" alt="Screen Shot 2020-02-16 at 5 15 02 PM" src="https://user-images.githubusercontent.com/59452077/74614728-05e4d280-50e0-11ea-8273-6636350f2fca.png">